frontend/src/modules/app/ToolBar.tsx

```diff
@@ -74,18 +80,35 @@ interface AlarmConfiguration {
   stateKey: string;
 }
 
+/**
+ * HeaderClock
+ *
+ * @component A container for displaying header clock of the page.
```

Comment:
What does "header clock of the page" mean? Is this container just the clock showing the current time at the right end of the toolbar?

Reply:
Yes its the component to show current time in toolbar